Formula/bash-snippets.rb

```diff
@@ -1,20 +1,53 @@
 class BashSnippets < Formula
   desc "Collection of small bash scripts for heavy terminal users"
   homepage "https://github.com/alexanderepstein/Bash-Snippets"
-  url "https://github.com/alexanderepstein/Bash-Snippets/archive/v1.16.0.tar.gz"
-  sha256 "19a029a204f16bd4db30cf7fa32aa6953dd10dcb996138b9bc65c01612afaac7"
+  url "https://github.com/alexanderepstein/Bash-Snippets/archive/v1.16.2.tar.gz"
+  sha256 "bbe899bf1b3095761fb405e9adebe9ff9bc4178dbcad2121a91d821b4308d107"
 
   bottle :unneeded
 
+  option "with-cheat", "Install cheat"
+  option "with-cloudup", "Install cloudup"
+  option "with-crypt", "Install crypt"
+  option "with-cryptocurrency", "Install cryptocurrency"
+  option "with-currency", "Install currency"
+  option "with-geo", "Install geo"
+  option "with-movies", "Install movies"
+  option "with-newton", "Install newton"
+  option "with-qrify", "Install qrify"
+  option "with-short", "Install short"
+  option "with-siteciphers", "Install siteciphers"
+  option "with-stocks", "Install stocks"
+  option "with-taste", "Install taste"
+  option "with-todo", "Install todo"
+  option "with-weather", "Install weather"
+  option "with-ytview", "Install ytview"
+  option "without-all-tools", "Do not install all available snippets"
+
+  conflicts_with "cheat", :because => "Both install a `cheat` executable" if build.with?("all-tools") || build.with?("cheat")
```

Comment:
ditto.
```
  if build.with?("all-tools") || build.with?("cheat")
    conflicts_with "cheat", :because => "Both install a `cheat` executable"
  end
```
and please squash this all down to 1 commit